sourcetool/pkg/policy/policy.go

```diff
@@ -235,31 +238,28 @@ func CreateLocalPolicy(ctx context.Context, ghconnection *ghcontrol.GitHubConnec
 	}
 
 	// Create the entire path if it doesn't already exist
-	if err := os.MkdirAll(filepath.Dir(path), 0770); err != nil {
+	if err := os.MkdirAll(filepath.Dir(path), 0o750); err != nil {
```

Comment:
Do we want to change the permissions?

Reply:
I don't know if they were correct in the first place, just curious.

Reply:
Any reason to have the directory group writable? I can silence the linter as with other perms if we need to.

Reply:
Nah, just wanted to make sure it was deliberate.